### PR TITLE
Download a folder

### PR DESCRIPTION
**Pull request by davivel**

### CODE REVIEW

**Comment by davivel** [comment]
@masensio , to review

File: src/com/owncloud/android/files/FileOperationsHelper.java [hunk 1]
```diff
@@ -197,18 +197,49 @@ public void sendDownloadedFile(OCFile file) {
...
+            Intent intent = new Intent(mFileActivity, OperationsService.class);
+            intent.setAction(OperationsService.ACTION_SYNC_FILE);
+            intent.putExtra(OperationsService.EXTRA_ACCOUNT, mFileActivity.getAccount());
+            intent.putExtra(OperationsService.EXTRA_REMOTE_PATH, file.getRemotePath());
+            intent.putExtra(OperationsService.EXTRA_SYNC_FILE_CONTENTS, true);
+            mWaitingForOpId = mFileActivity.getOperationsServiceBinder().queueNewOperation(intent);
+            mFileActivity.showLoadingDialog();
+            
+        } else {
+            /*
```
Comment (masensio) [inline src/com/owncloud/android/files/FileOperationsHelper.java]: Please, remove commented code if it is unnecessary for future features.

File: src/com/owncloud/android/files/services/FileDownloader.java [hunk 2]
```diff
@@ -130,52 +120,86 @@ public void onCreate() {
...
-            Message msg = mServiceHandler.obtainMessage();
-            msg.arg1 = startId;
-            msg.obj = requestedDownloads;
-            mServiceHandler.sendMessage(msg);
+        } else {
+            final Account account = intent.getParcelableExtra(EXTRA_ACCOUNT);
+            final OCFile file = intent.getParcelableExtra(EXTRA_FILE);
+
+            /*Log_OC.v(
+                    "NOW " + TAG + ", thread " + Thread.currentThread().getName(),
```
Comment (masensio) [inline src/com/owncloud/android/files/services/FileDownloader.java]: Remove commented code.

File: src/com/owncloud/android/files/FileOperationsHelper.java (hunk 1 above)
Comment (davivel) [inline src/com/owncloud/android/files/FileOperationsHelper.java]: I will.

File: src/com/owncloud/android/files/services/FileDownloader.java (hunk 2 above)
Comment (davivel) [inline src/com/owncloud/android/files/services/FileDownloader.java]: I will remove the ```ACTION_CANCEL_FILE```, but I'd rather leave the logs.

**Comment by masensio** [comment]
@davivel  reviewed

**Pull request merged**